production.env

```diff
@@ -68,13 +59,13 @@ FTPS_PASSIVE_PORTS=8100-8399
 
 # Various helpers
 ADMIN_EMAIL="MiG Info <mig@migrid.test>"
-ADMIN_LIST=
-SMTP_SENDER=
+ADMIN_LIST="/C=DK/ST=NA/L=NA/O=EX/OU=NA/CN=John Doe/emailAddress=jd@example.com"
+SMTP_SENDER="Do Not Reply <no-reply@example.com>"
```

Comment:
Not sure if we should use @migrid.test here as well

Reply:
Alright, I don't disagree about the option to have external admins. 
It was just that the previous line (hidden above) does have exactly that site connection  
`ADMIN_EMAIL="MiG Info <mig@migrid.test>"`
and that one generally shouldn't send out mail with a spoofed domain like that `SMTP_SENDER` setting will do :)